app/assets/stylesheets/clients.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the clients controller here.
```

Comment:
Pls removed the unused files.

Reply:
removed.

Reply:
There are two files in the application_cable folder should I remove these too?